### PR TITLE
fix: remove double spaces in i18n

### PR DESCRIPTION
**Pull request by areski**

### CODE REVIEW

File: registration/templates/registration/password_reset_complete.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <p>
     {% trans "Your password has been reset!" %}
-    {% trans "You may now" %}<a href="{{ login_url }}">{% trans "log in" %}</a>.
+    {% trans "You may now" %} <a href="{{ login_url }}">{% trans "log in" %}</a>.
```
Comment (dicato) [inline registration/templates/registration/password_reset_complete.html]: I think you may need to add a single white space in this sentence.

Comment (areski) [inline registration/templates/registration/password_reset_complete.html]: I was trying to not change the existing gettext translation, to my opinion would be best for tiny change like thisone.

If the maintainer wants to change then maybe we could take this approach:
{% blocktrans with l_url=login_url %}You may now {{ l_url }} log in{% endblocktrans %}.

Comment (dicato) [inline registration/templates/registration/password_reset_complete.html]: My experience with translations in Django is limited, but after a quick refresher with the documentation I think this is a great approach.

I'd suggest making the change you suggested as a separate commit. I'll happily do a review and merge.

Comment (areski) [inline registration/templates/registration/password_reset_complete.html]: Me == confuse
Are you the package maintainer!? I thought it was @macropin

File: registration/templates/registration/password_reset_complete.html [hunk 2]
```diff
@@ -6,7 +6,8 @@
 {% block content %}
 <p>
     {% trans "Your password has been reset!" %}
-    {% trans "You may now" %}<a href="{{ login_url }}">{% trans "log in" %}</a>.
+    {% blocktrans %}You may now <a href="{{ login_url }}">log in</a>{% endblocktrans %}.
```
Comment (areski) [inline registration/templates/registration/password_reset_complete.html]: @dicato here the change we just mentioned

File: registration/templates/registration/password_reset_complete.html (hunk 1 above)
Comment (dicato) [inline registration/templates/registration/password_reset_complete.html]: I'm a collaborator. :smile: @macropin felt comfortable with giving me collaborator privileges after contributing a couple pull requests. I use django-registration-redux in production and decided to take a more active approach after being underwhelmed with user registration options in Django.

Comment (areski) [inline registration/templates/registration/password_reset_complete.html]: I'm glad to hear that, you should put both of your names on the README
So ppl knows who is maintaining

File: registration/templates/registration/password_reset_complete.html (hunk 2 above)
Comment (dicato) [inline registration/templates/registration/password_reset_complete.html]: :+1:

**Comment by dicato** [comment]
@areski thank you! Merging!

**Pull request merged**